extensions/kubectl-cli/types/shell-path.d.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
```

Comment:
maybe it should in the `types` folder at the root ?

Reply:
extensions need to be 'independent'